components/shared/components/Widget/components/panes/DonorPane/DonorPane.tsx

```diff
@@ -301,18 +304,14 @@ export const DonorPane: React.FC<{ text: WidgetPane2Props }> = ({ text }) => {
             </div>
 
             <RadioButtonGroup
-              options={[
-                {
-                  title: text.payment_method_selector_bank_text,
-                  value: PaymentMethod.BANK,
-                  data_cy: "bank-method",
-                },
-                {
-                  title: text.payment_method_selector_vipps_text,
-                  value: PaymentMethod.VIPPS,
-                  data_cy: "vipps-method",
-                },
-              ]}
+              options={paymentMethods.map((method) => ({
+                title: method.text,
+                value: method.id,
+                data_cy: {
```

Comment:
Since the payment method id's are fairly static (they come from the database), could we redo this data_cy thing to simply use the method id directly?

This seems to be a very complicated way of solving the problem, if we could simply do 
```javascript
(method) => ({
  title: method.text,
  value: method.id,
  data_cy: `method-${method.id}`
})
```